evap/staff/tests/test_views.py

```diff
@@ -1303,6 +1304,24 @@ def test_view_downloads_csv_file(self):
         self.assertEqual(response.content, expected_content.encode("utf-8"))
 
 
+class TestSemesterVoteTimestampsExport(WebTestStaffMode):
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.timestamp = baker.make(VoteTimestamp)
+
+        cls.url = f"/staff/semester/{cls.timestamp.evaluation.course.semester.pk}/vote_timestamps_export"
+        cls.test_users = [cls.manager]
+
+    def test_view_downloads_csv_file(self):
+        response = self.app.get(self.url, user=self.manager)
+        expected_content = (
+            "Evaluation id;Timestamp;Course type;Course degrees;Vote end date\n"
+            + f"{self.timestamp.evaluation.id};{self.timestamp.timestamp};{self.timestamp.evaluation.course.type.name};{', '.join([degree.name for degree in self.timestamp.evaluation.course.degrees.all()])};{self.timestamp.evaluation.vote_end_datetime}\n"
+        ).encode("utf-8")
```

Comment:
This is just the code from the view again, wouldn't it be better to specify values when creating the timestamp / evaluation and check for those values to appear again here? @He3lixxx